src/service_router.rs

```diff
@@ -147,7 +147,7 @@ describe! service_router {
                             &service_router).unwrap();
 
             let result = response::extract_body_to_string(response);
-            assert_eq!(result, "No Such Service: unknown-id");
+			assert_eq!(result, "{\"error\":\"NoSuchService\",\"id\":\"unknown-id\"}");
```

Comment:
right, I got it wrong. See https://github.com/fxbox/foxbox/blob/master/src/utils.rs#L72